### ISSUE TITLE
"DEPRECATION WARNING: Initialization autoloaded the constants ActionText::ContentHelper and ActionText::TagHelper" on Rails 6.1+

### ISSUE THREAD

**Issue by xxx** [pboling/sanitize_email#66]
Hi, thank you for making this gem.

When used on Rails 6.1 and later, this gem causes the following warning at startup:

```
DEPRECATION WARNING: Initialization autoloaded the constants ActionText::ContentHelper and ActionText::TagHelper.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload ActionText::ContentHelper, for example,
the expected changes won't be reflected in that stale Module object.

These autoloaded constants have been unloaded.

In order to autoload safely at boot time, please wrap your code in a reloader
callback this way:

    Rails.application.reloader.to_prepare do
      # Autoload classes and modules needed at boot time here.
    end

That block runs when the application boots, and every time there is a reload.
For historical reasons, it may run twice, so it has to be idempotent.

Check the "Autoloading and Reloading Constants" guide to learn more about how
Rails autoloads and reloads.
 (called from <top (required)> at /home/mpd/git-sources/catchandrelease/config/environment.rb:7)
```

The actual issue is probably the reference to `ActionController::Base` in [engine.rb](https://github.com/pboling/sanitize_email/blob/master/lib/sanitize_email/engine.rb#L10). 

The rails-side issue regarding this warning (with a lot more context) is [here](https://github.com/rails/rails/issues/36546).

**Comment by pboling** [comment]
Thanks for the report.  I'll see if it repro's in Rails 7 and fix.  If it is Rails 6 only, then I'll let it go.

**Comment by pboling** [comment]
Fixed by 4276069
